### PR TITLE
Add SSLBL package

### PR DESCRIPTION
**Pull request by mavam**
This PR adds an [SSLBL](https://sslbl.abuse.ch/) package, which came to life while [writing a tutorial on how to create package](https://github.com/tenzir/tenzir/pull/4568).

### CODE REVIEW

File: sslbl/package.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  update-context:
+    name: Update SSLBL Context
+    description: |
+      A pipeline that periodically refreshes the SSLBL lookup table.
+    definition: |
+      // tql2
+      every {{ inputs.refresh-interval }} {
+        load_http "from https://sslbl.abuse.ch/blacklist/sslblacklist.csv"
+        read_csv comments=true, header="timestamp,SHA1,reason"
+        legacy "context update sslbl --key=SHA1"
```
Comment (dominiklohmann) [inline sslbl/package.yaml:36]: Ideally, keep as few operators as possible in `every`. The `context update` can and should be moved out.

File: sslbl/package.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+id: sslbl
+name: SSLBL
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: |
+  https://raw.githubusercontent.com/tenzir/library/main/sslbl/package.svg
+description: |
+  The [SSLBL](https://sslbl.abuse.ch/) package makes available a lookup table
```
Comment (dominiklohmann) [inline sslbl/package.yaml:8]:
```suggestion
  The [SSL Blocklist (SSLBL)](https://sslbl.abuse.ch/) package makes available a lookup table
```

File: sslbl/package.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+inputs:
+  refresh-interval:
+    name: Time between context updates
+    description: |
+      The duration between updates that fetch the SSLBL database via the API.
+    default: 1 hour
+
+contexts:
+  feodo:
```
Comment (dominiklohmann) [inline sslbl/package.yaml:20]: Wrong context name—did you test this package locally?

File: sslbl/package.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+id: sslbl
+name: SSLBL
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: |
+  https://raw.githubusercontent.com/tenzir/library/main/sslbl/package.svg
+description: |
+  The [SSLBL](https://sslbl.abuse.ch/) package makes available a lookup table
+  with SHA1 hashes of blacklisted certificates that can be used when monitoring
+  SSL/TLS certificate exchanges.
```
Comment (dominiklohmann) [inline sslbl/package.yaml:10]: I think the description should mention the context name.

File: sslbl/package.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  - name: Enrich Suricata TLS logs with SSLBL domains
+    description: |
+      Enriches the certificate SHA1 fingerprint from Suricata TLS logs with the
+      SSLBL data.
+    definition: |
+      // tql2
+      subscribe "suricata"
+      | where @name == "suricata.tls"
+      | set sha1 = tls.fingerprint.replace(":", "")
+      | legacy "enrich sha1 sslbl"
```
Comment (dominiklohmann) [inline sslbl/package.yaml:50]:
```suggestion
      where @name == "suricata.tls"
      sha1 = tls.fingerprint.replace(":", "")
      legacy "enrich sha1 sslbl"
```

While not incorrect, let's use a consistent style with the other packages.

File: sslbl/package.yaml (hunk 3 above)
Comment (mavam) [reply sslbl/package.yaml:20]: Yes, but I ended up here: https://github.com/tenzir/tenzir/pull/4568#discussion_r1749166210

**Review by dominiklohmann — APPROVED** [review]

**Pull request merged**